src/ontology/run.sh

```diff
@@ -22,4 +22,4 @@ if [ x$ODK_DEBUG = xyes ]; then
     TIMECMD="/usr/bin/time -f ### DEBUG STATS ###\nElapsed time: %E\nPeak memory: %M kb"
 fi
 
-docker run -v $PWD/../../:/work -w /work/src/ontology -e ROBOT_JAVA_ARGS='-Xmx18G' -e JAVA_OPTS='-Xmx18G' --rm -ti obolibrary/$IMAGE $TIMECMD "$@"
```

Comment:
off to see this here and not being reflected by phenio-odk.yaml!

Reply:
It is in the phenio-odk.yaml though - https://github.com/monarch-initiative/phenio/blob/2a8f0cacdd797a0bc78fe1343ae329392dd4498d/src/ontology/phenio-odk.yaml#L93

Reply:
oh ok. Someone must have hacked thir previously :P